extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -595,7 +653,52 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
               visiblePercentageMin: 50,
               continuousTimeMin: 1,
             });
-    return superReturnValue;
+    return frameLoadPromise;
+  }
+
+  /**
+   * Performs the necessary measuring and layout steps needed to render fluid
+   * creatives. This includes computing the width of the ad slot's parent
+   * element as well as styling the ad slot's width to match, assuming no width
+   * for the slot has been provided.
+   */
+  measureAndLayoutForFluid_() {
+    if (!this.element.style.width) {
+      this.getVsync().run({
```

Comment:
I no longer think you need the vsync mutate as this now runs in the same execution as layoutCallback

Reply:
Done.

Reply:
No longer needed; got things working by styling everything through CSS now. Not sure why I wasn't able to do it before.